cognite/neat/rules/models/entities.py

```diff
@@ -0,0 +1,299 @@
+import re
```

Comment:
Is this renamed from `_entity.py` from `models` ?

Reply:
No, I made a new parallell to avoid breaking existing, then I will spend the next PR moving over to these new entities.